packages/nextjs/components/scaffold-eth/TxnToast.tsx

```diff
@@ -0,0 +1,12 @@
+export default function TxnToast({ message, blockExplorerLink }: { message: string; blockExplorerLink?: string }) {
```

Comment:
This is just an idea:

How would you feel about moving this inside the `useTransactor` hook? Since we are only using it there (and you are supposed to always use the transactor for sending tx) we could declare it above the `useTransactor` declaration.

Hooks / components that use a custom JSX content could declare it in the same file (the same thing we do with types).

Anyway, up to you!